### PR TITLE
LPS-153687 Add fetchSharingCollaborators to the Info display Context

### PR DESCRIPTION
**Pull request by cgoncas**
### Motivation

As a content marketer, I want to see all my collaborators after sharing an asset, so that I can easily view and manage my collaborators on a specific asset.

### Proposed Solution

This solution use the same approach described in https://github.com/liferay-tango/liferay-portal/pull/2612.

We have created a sharing_collaborators.jsp that includes the sharing:collaborators taglib, so the sharing collaborators is rendered using a fetch, as we do with the content performance panel.

Also in this PR we have include the new actions for Blogs and FileEntries that create a URL to this new JSP.

### How to test it

Until the frontend part is send, no functional tests can be executed.

Integration tests will be sent in another PR.

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:forward

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java [hunk 1]
```diff
@@ -142,6 +142,10 @@ protected void doServeResource(
 					"fetchSharingButtonURL",
 					_getFetchSharingButtonURL(
 						contentDashboardItem, httpServletRequest)
+				).put(
+					"fetchSharingContactsURL",
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java:146]: hi @cgoncas !
you referenced [here](https://github.com/liferay-tango/liferay-portal/pull/2626#issuecomment-1195749318) `fetchSharingCollaboratorsURL` but I read here `fetchSharingContactsURL`. Which one should we use finally? As you said `fetchSharingCollaboratorsURL` makes more sense

Comment (cgoncas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java:146]: fetchSharingCollaboratorsURL should be the right one :), could you fix it :)

Comment (beltranrengifo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java:146]: Right, I'll fix it, thxs!

File: modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/action/SharingCollaboratorsFileEntryContentDashboardItemAction.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+			PortletResponse portletResponse =
+				(PortletResponse)_httpServletRequest.getAttribute(
+					JavaConstants.JAVAX_PORTLET_RESPONSE);
+
+			return PortletURLBuilder.createRenderURL(
+				_portal.getLiferayPortletResponse(portletResponse),
+				"com_liferay_content_dashboard_web_portlet_" +
+					"ContentDashboardAdminPortlet"
+			).setMVCPath(
+				"/sharing_collaborators_button.jsp"
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/action/SharingCollaboratorsFileEntryContentDashboardItemAction.java:82]: @cgoncas maybe we missed to rename this path to `"/sharing_collaborators.jsp"`
i'm getting an error I did not realized before:

```
javax.portlet.PortletException: Path /sharing_collaborators_button.jsp is not accessible by portlet com_liferay_content_dashboard_web_portlet_ContentDashboardAdminPortlet
```

thxs!

**Pull request closed without merging**